ts/config.ts

```diff
@@ -4,6 +4,7 @@ import Config from "react-native-config";
 
 export const environment = Config.ENVIRONMENT;
 export const apiUrlPrefix = Config.API_URL_PREFIX;
+export const apiDefaultTimeoutMs = parseInt(Config.API_DEFAULT_TIMEOUT_MS, 10);
```

Comment:
mmh this is bad, this could throw an exception, at runtime, if we make a mistake and deploy an app with a bad config, the app will crash at start, better catch the exception and have a default value